<ui version="4.0" >
 <class>SceneModelerWidget</class>
 <widget class="QWidget" name="SceneModelerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Number of discretization steps</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="sliderDSteps" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QSpinBox" name="numdstepsEdit" >
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Threshold for scene detection</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QDoubleSpinBox" name="epsilon" >
     <property name="singleStep" >
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>131</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2" >
    <widget class="QPushButton" name="computeButton" >
     <property name="text" >
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="progressLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderDSteps</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numdstepsEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>274</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel" >
     <x>413</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numdstepsEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderDSteps</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>405</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel" >
     <x>283</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
